Makefile

```diff
@@ -3,9 +3,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
 GO_CMD ?= go
+# Controller API entry point to be used for generating REST API specifications
+API_SPEC_META=cmd/aries-agentd/main.go
```

Comment:
Same comment here. Be explicit on the type of API.

Reply:
done